book/src/ui/interacting.md

```diff
@@ -0,0 +1,159 @@
+# Interaction!
+
+Now that we have a button we can interact with, let's see how we actually do this.
+We will show you how to do this in two ways. One way will be interaction through a system,
+and the latter interaction through `handle_event` method of your active state.
+
+## Creating the system
+
+Let's start of with some boilerplate code: 
+
+```rust 
+pub struct SimpleButtonSystem;
+
+impl System for SimpleButtonSystem {
+	type SystemData = ();
+
+	fn run(&mut self, data: Self::SystemData) {
+	
+	}
+}
+```
+
+This was shown in previous chapters. The way you will be able to read the generated 
+events is with a [ReaderId](https://docs.amethyst.rs/master/specs/prelude/struct.ReaderId.html).
+The `ReaderId` is added as a field to the system struct.
+
+The events we want to read are of type [UiEvent](https://docs.amethyst.rs/master/amethyst_ui/struct.UiEvent.html).
+We also need to fetch the [EventChannel](https://docs.amethyst.rs/master/shrev/struct.EventChannel.html) in our `SystemData`, 
+since the `ReaderId` actually pulls (reads) information  from the `EventChannel`.
+
+Adding it up, it should look like this: 
+
+```rust
+pub struct SimpleButtonSystem {
+	reader_id: ReaderId<UiEvent>,
```

Comment:
best use 4 spaces instead of tabs to indent

also, there's missing imports so the build wouldn't pass. at the top of my mind:

```rust
# extern crate amethyst;
#
# use amethyst::ecs::System;
# use amethyst::shrev::{EventChannel, ReaderId};
# use amethyst::ui::UiEvent;
```